addons/website_customer/views/website_customer_templates.xml

```diff
@@ -107,10 +107,10 @@
                                 <div class="o_wc_offcanvas_tags accordion-collapse collapse">
                                     <div class="accordion-body">
                                         <div class="d-flex flex-wrap align-items-center gap-1 mb-4" t-if="len(tags)">
-                                            <a class="me-2" t-attf-href="/customers/#{ current_industry_id and 'industry/%s/' % slug(current_industry) or '' }#{ current_country_id and 'country/%s' % slug(current_country) or '' }">
+                                            <a class="badge text-bg-light" t-attf-href="/customers/#{ current_industry_id and 'industry/%s/' % slug(current_industry) or '' }#{ current_country_id and 'country/%s' % slug(current_country) or '' }">
```

Comment:
For reference, before the change it looks out of place
![image](https://github.com/odoo/odoo/assets/43653136/eb9fd893-2f4e-4d9d-b910-5c6ea7a0c9cd)
vs after
![image](https://github.com/odoo/odoo/assets/43653136/1bc2e179-70ae-43c1-b5f0-9f18e0fd5ccc)

Reply:
Thanks for the review
Do you think you can check this @tde-banana-odoo and eventually R+ it ? 
Cheers !